api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -765,6 +765,21 @@ message HttpConnectionManager {
   // If this config is set, the Proxy-Status HTTP response header field is
   // populated. By default, it is not.
   ProxyStatusConfig proxy_status_config = 49;
+
+  // Configuration options for Unified Header Validation (UHV).
+  // UHV is an extensible mechanism for checking validity of HTTP requests as well as providing
+  // normalization for request attributes, such as URI path.
+  // The default UHV checks the following:
+  // 1. H/1 header map validity according to `RFC 7230 section 3.2<https://tools.ietf.org/html/rfc7230#section-3.2>`_
```

Comment:
I love H/1 and H/2 but I've seen folks confused enough I think we want to spell them out.

Reply:
Done